### PR TITLE
added subtraction

### PR DESCRIPTION
**Pull request by Seth-Park**
added subtraction

### CODE REVIEW

File: calc.py [hunk 1]
```diff
@@ -1,2 +1,5 @@
 def add(a, b):
     return a + b
+
+def subtract(a, b):
+    return a - b
```
Comment (AceHao) [inline calc.py]: !!!!!

File: calc.py [hunk 2]
```diff
@@ -1,2 +1,5 @@
 def add(a, b):
     return a + b
+
+def subtract(a, b):
```
Comment (stefanv) [inline calc.py]: Excellent.  Can you add a docstring?

**Comment by jenniferdai1** [comment]
looks good

**Comment by Seth-Park** [comment]
Made fixes, review please!

**Comment by 18epedersen** [comment]
:100:

**Comment by AceHao** [comment]
Nice!

**Comment by stefanv** [comment]
Conflicts--can you please rebase on master?

**Pull request merged**